### ISSUE TITLE
iPad app

### ISSUE THREAD

**Issue by nreilly** [keybase/client#10006]
Please create an iPad app, with all the lovely features like split view, supporting the iPad Pro Smart Keyboard well (navigate around with keyboard, return key to send messages, etc)

**Comment by specious** [comment]
Popular request - #9082, #8909, #8902, #6230

See: #6836

**Comment by sergioabril** [comment]
Yes please, add iPad support. I’d be another step into replacing Slack (at least for me)

**Comment by specious** [comment]
I'm constantly turning my iPad sideways to send a Keybase chat message. Is living in the future always supposed to be this awkward?

**Comment by elks** [comment]
I would love to see an iPad app added.

**Comment by IpsmLorem** [comment]
Please

**Comment by BergQuester** [comment]
Dupe of #8909 which as of writing has an open pull request. The contributor is open to implementing a split view as well.

**Comment by richardsj** [comment]
Focus on #8909 folks, this is ridiculous
